### PR TITLE
Make pulp startup timeout variable

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

**Review by mnasiadka — COMMENTED** [review]
Wouldn't it be better to increase delay?

**Comment by Alex-Welsh** [comment]
> Wouldn't it be better to increase delay?

Why would that be preferable?

**Comment by mnasiadka** [comment]
> > Wouldn't it be better to increase delay?
> 
> Why would that be preferable?

Less cycles to use ;-) I'm not a fan of hammering a service every 2 seconds for a long time. We could also make it configurable, because the commit message doesn't say why we need it.

**Comment by Alex-Welsh** [comment]
Reworked to make the delay user configurable

File: etc/kayobe/containers/pulp/post.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@
     url: "{{ pulp_url }}/pulp/api/v3/status/"
   register: pulp_status
   until: pulp_status is success
-  retries: 30
-  delay: 2
+  retries: "{{ pulp_timeout_retries | default(30) }}"
+  delay: "{{ pulp_timeout_delay | default(2) }}"
```
Comment (markgoddard) [inline etc/kayobe/containers/pulp/post.yml]: Let's increase the default enough to fix your problem @Alex-Welsh

Comment (Alex-Welsh) [reply etc/kayobe/containers/pulp/post.yml]: Done

**Review by markgoddard — APPROVED** [review]

**Pull request merged**